infra/load-balancer.tf

```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "load_balancer_controller" {
+  name       = "jovand-loadbalancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
```

Comment:
Make sure you add specific version of the chart as well so next time you run apply it doesn't change.